### PR TITLE
First cut for 02-05

### PR DESCRIPTION
**Pull request by kaarthis**
Please review this.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Starting with Kubernetes 1.26:
+  * HostProcess Containers will be GA
+  * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Starting with Kubernetes 1.27:
+  * The Max Surge default value will change on newly created nodepools from 1 to 10%.
```
Comment (shashankbarsin) [inline CHANGELOG.md:13]: should this be under behavior change section?

Comment (shashankbarsin) [reply CHANGELOG.md:13]: and also can we add link to doc

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+* Starting with Kubernetes 1.27:
+  * The Max Surge default value will change on newly created nodepools from 1 to 10%. 
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+* Azure Policy will be updated to [GateKeeper 3.11](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.11.0) on Feb 20th for AKS 1.24 and up.
+* Workload Identity: Application pods using workload identity will need the following label added `azure.workload.identity/use` starting with the 2023-01-29 release. Add the label to your running pods/deployments to avoid pods from failing at restart. See more [here](https://learn.microsoft.com/azure/aks/workload-identity-overview#service-account-labels).
```
Comment (shashankbarsin) [inline CHANGELOG.md:16]: do we still need this line since it talks about 01-29 release?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+
+### Release notes
+
+* Features
+  * Managed Prometheus addon image release. See [release notes](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-01-31-2023)
+  * AKS introduces a new [Standard tier](https://learn.microsoft.com/azure/aks/free-standard-pricing-tiers) which includes the previous standalone uptime SLA in addition to improved capabilities over the Free tier. Read the [blog(https://aka.ms/aks/standard-tier-blog) to learn more about the launch of the Standard tier. Azure API is updated to include the new “Standard” tier, as a result, "Basic" and "Paid" will be removed in the 2023-07-01 API version, and this will be a breaking change in API version 2023-07-01 or newer. If you use automated scripts, CD pipelines, ARM templates, Terraform, or other third-party toolings that rely on the above parameters, please be sure to make the necessary changes before upgrading to the 2023-07-01 or newer API version. From API version 2023-01-01 and newer, you can start transitioning to the new API parameters "Base" and "Standard".
```
Comment (shashankbarsin) [inline CHANGELOG.md:22]: missing ']' after '[blog'

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+* Bug Fix
+  * [HTTP Proxy](https://learn.microsoft.com/en-us/azure/aks/http-proxy##updating-proxy-configurations) Fixed an issue on the "No Proxy" update -  where the cluster FQDN would be removed from noProxy on updates.
```
Comment (shashankbarsin) [inline CHANGELOG.md:24]: we need to remove 'en' to be localization friendly

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,36 @@
...
+* Bug Fix
+  * [HTTP Proxy](https://learn.microsoft.com/en-us/azure/aks/http-proxy##updating-proxy-configurations) Fixed an issue on the "No Proxy" update -  where the cluster FQDN would be removed from noProxy on updates. 
+  * The aks swagger api specs now moved under a subfolder per the [issue](https://github.com/Azure/azure-sdk/issues/5385)
+* Component Updates
+  * Add support for [defender agent](https://learn.microsoft.com/en-us/azure/defender-for-cloud/defender-for-kubernetes-introduction#what-are-the-benefits-of-microsoft-defender-for-kubernetes) to run on FIPS machines.
```
Comment (shashankbarsin) [inline CHANGELOG.md:27]: we need to remove 'en-us'

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,36 @@
...
+* Bug Fix
+  * [HTTP Proxy](https://learn.microsoft.com/en-us/azure/aks/http-proxy##updating-proxy-configurations) Fixed an issue on the "No Proxy" update -  where the cluster FQDN would be removed from noProxy on updates. 
+  * The aks swagger api specs now moved under a subfolder per the [issue](https://github.com/Azure/azure-sdk/issues/5385)
```
Comment (shashankbarsin) [inline CHANGELOG.md:25]: This is not a bug fix. This should be under announcements section IMO

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,36 @@
...
+* Component Updates
+  * Add support for [defender agent](https://learn.microsoft.com/en-us/azure/defender-for-cloud/defender-for-kubernetes-introduction#what-are-the-benefits-of-microsoft-defender-for-kubernetes) to run on FIPS machines.
+  * Clients (e.g. portal / CLI / powershell) can now discover the trusted access role bindings operations on available operations..
```
Comment (shashankbarsin) [inline CHANGELOG.md:28]: there's an additional '.' at the end of the line

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,36 @@
...
+  * The Max Surge default value will change on newly created nodepools from 1 to 10%. 
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+* Azure Policy will be updated to [GateKeeper 3.11](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.11.0) on Feb 20th for AKS 1.24 and up.
+* Workload Identity: Application pods using workload identity will need the following label added `azure.workload.identity/use` starting with the 2023-01-29 release. Add the label to your running pods/deployments to avoid pods from failing at restart. See more [here](https://learn.microsoft.com/azure/aks/workload-identity-overview#service-account-labels).
+
+### Release notes
+
+* Features
+  * Managed Prometheus addon image release. See [release notes](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-01-31-2023)
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]: This is just an updated set of images for managed prometheus right or are there some new features? if it's the former, we need to move this under component updates section

Comment (kaarthis) [reply CHANGELOG.md]: its former i ll have it moved.

File: CHANGELOG.md (hunk 2 above)
Comment (kaarthis) [reply CHANGELOG.md:16]: it will get dynamically updated with every release so the release tracker is a must at the top.

**Review by kaarthis — COMMENTED** [review]
Please review

File: CHANGELOG.md (hunk 1 above)
Comment (kaarthis) [reply CHANGELOG.md:13]: We dont have a doc yet but we will soon have it.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,36 @@
...
+* The aks swagger api specs now moved under a subfolder per the [issue](https://github.com/Azure/azure-sdk/issues/5385).
+
+### Release notes
+
+* Features
+  * AKS introduces a new [Standard tier](https://learn.microsoft.com/azure/aks/free-standard-pricing-tiers) which includes the previous standalone uptime SLA in addition to improved capabilities over the Free tier. Read the [blog](https://aka.ms/aks/standard-tier-blog) to learn more about the launch of the Standard tier. Azure API is updated to include the new “Standard” tier, as a result, "Basic" and "Paid" will be removed in the 2023-07-01 API version, and this will be a breaking change in API version 2023-07-01 or newer. If you use automated scripts, CD pipelines, ARM templates, Terraform, or other third-party toolings that rely on the above parameters, please be sure to make the necessary changes before upgrading to the 2023-07-01 or newer API version. From API version 2023-01-01 and newer, you can start transitioning to the new API parameters "Base" and "Standard".
```
Comment (wangyira) [inline CHANGELOG.md:22]: should this go into announcements or features @kaarthis?

**Review by wangyira — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 9 above)
Comment (kaarthis) [reply CHANGELOG.md:22]: yeah good point should be announcements i guess.

**Review by qpetraroia — APPROVED** [review]
I still need to add some things for KEDA, but approving in the meantime. Looks good!

**Review by wangyira — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,34 @@
...
+* Azure Policy will be updated to [GateKeeper 3.11](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.11.0) on Feb 20th for AKS 1.24 and up.
+* Workload Identity: Application pods using workload identity will need the following label added `azure.workload.identity/use` starting with the 2023-01-29 release. Add the label to your running pods/deployments to avoid pods from failing at restart. See more [here](https://learn.microsoft.com/azure/aks/workload-identity-overview#service-account-labels).
+* The aks swagger api specs now moved under a subfolder per the [issue](https://github.com/Azure/azure-sdk/issues/5385).
+
+### Release notes
+
+* Bug Fix
+  * [HTTP Proxy](https://learn.microsoft.com/azure/aks/http-proxy##updating-proxy-configurations) Fixed an issue on the "No Proxy" update -  where the cluster FQDN would be removed from noProxy on updates.
```
Comment (miwithro) [inline CHANGELOG.md:22]: Already announced the http proxy "no_proxy" fix on last weeks release notes.  You can remove this item.

**Pull request merged**